index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
+[=triggered by user activation|user triggered activation=], may be used as a
+[=new information about the user's intent|criteria=] to grant access to [=sensor readings|readings=].
```

Comment:
@jyasskin Is user gesture considered to be 'information about user's intent'? Maybe I should rephrase text somehow. We already have hook to Permission API for acquiring access to sensor reading, I just wanted to add note for user gesture as one of the intents that UA might use for e.g., auto-granting or prompting user.